src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/HaFlowExtended.groovy

```diff
@@ -284,4 +294,51 @@ class HaFlowExtended {
                 .build();
         return new HaFlowBidirectionalExam(traffExam, forward1, forward2, reverse1, reverse2);
     }
+
+    /**
+     * This check allows us to perform main HA-Flow properties comparison after updating operations
+     * Note, some fields such as timeUpdate, status,, subFlows.timeUpdate, subFlows.status, subFlows.forwardLatency,
+     * subFlows.reverseLatency, subFlows.latencyLastModifiedTime are excluded for verification.
```

Comment:
I'd add to explanation that the fields are excluded from verification because we can't predict their values in environment.